torch/_inductor/fx_passes/split_cat.py

```diff
@@ -781,7 +781,7 @@ def get_transform_params(
         split_dim = unbind_node.kwargs["dim"]
         transform_params_list = []
         for user_node, user_inputs in zip(next_users, user_inputs_list):
-            cat_dim = get_arg_value(user_node, 1, "dim")
+            cat_dim = get_arg_value(user_node, 1, "dim") or 0
```

Comment:
why was this necessary?

Reply:
This fixed an existing bug, not relevant to this change directly.